uitools/examples/cpp_quick/src/main.cpp

```diff
@@ -35,8 +35,6 @@
 //------------------------------------------------------------------------------
 int main(int argc, char* argv[])
 {
-  // Enforce OpenGL
-  qputenv("QSG_RHI_BACKEND", "opengl");
```

Comment:
This is unrelated changes I missed in an earlier round of changes. We no longer need to force OpenGL for the toolkit examples that are not AR-specific.